src/types.ts

```diff
@@ -281,120 +350,80 @@ export interface Tx {
 }
 
 /**
- * Array of transactions
- * @see {@link Tx}
+ * Array of OldTxs for Admin (Deprecated)
  */
-export interface Txs extends Array<Tx> {}
+export interface OldTxs extends Array<OldTx> {}
 
 /**
- * MAP protocol definition
+ * Utxo interface for non-admin (User) endpoints.
  */
-export interface MapProtocol {
-  /**
-   * App name
-   */
-  app?: string;
-  /**
-   * Type of MAP action
-   */
-  type?: string;
-  /**
-   * MAP key value pairs
-   */
-  keys?: { [key: string]: any };
+export interface Utxo {
+  id: string;
+  xpubId: string;
+  satoshis: number;
+  scriptPubKey: string;
+  type: string;
+  draftId?: string;
+  reservedAt?: Date;
+  spendingTxId?: string;
+  transaction?: Tx;
+  createdAt: Date;
+  updatedAt?: Date;
+  deletedAt?: Date;
 }
 
 /**
- * OP_RETURN data, only one of the attributes should be set
+ * Array of Utxos for non-admin (User) endpoints.
  */
-export interface OpReturn {
-  /**
-   * HEX string of OP_RETURN
-   */
-  hex?: string;
-  /**
-   * Array of parts of the OP_RETURN in hex
-   */
-  hex_parts?: string[];
-  /**
-   * MAP protocol definition
-   * @see {@link Map}
-   */
-  map?: MapProtocol;
-  /**
-   * String parts array
-   */
-  string_parts?: string[];
-}
+export interface Utxos extends Array<Utxo> {}
 
 /**
- * Recipient interface
+ * Old Utxo interface for Admin endpoints (Deprecated)
  */
-export interface Recipient {
-  to: string;
+export interface OldUtxo {
+  id: string;
+  xpub_id: string;
   satoshis: number;
-  op_return?: OpReturn;
+  script_pub_key: string;
+  type: string;
+  draft_id?: string;
+  reserved_at?: Date;
+  spending_tx_id?: string;
+  transaction?: OldTx;
+  created_at: Date;
+  updated_at?: Date;
+  deleted_at?: Date;
 }
 
 /**
- * Array of Recipients
- * @see {@link Recipient}
- */
-export interface Recipients extends Array<Recipient> {}
-
-/**
- * Fee unit to use when calculating the fee for the transaction (satoshis per byte)
+ * Array of OldUtxos for Admin (Deprecated)
  */
-export interface FeeUnit {
-  /**
-   * Satoshis
-   */
-  satoshis: number;
-  /**
-   * Bytes
-   */
-  bytes: number;
-}
+export interface OldUtxos extends Array<OldUtxo> {}
 
 /**
- * A record pointing to a UTXO by transaction ID and output index
+ * Paymail address interface for non-admin (User) endpoints.
  */
-export interface UtxoPointer {
-  /**
-   * Transaction ID
-   */
-  transaction_id: string;
-  /**
-   * Output index
-   */
-  output_index: number;
+export interface PaymailAddress {
+  id: string;
+  xpubId: string;
+  alias: string;
+  domain: string;
+  publicName: string;
```

Comment:
Just for info: public name in `CreatePaymail` looks like this
`PublicName string json:"public_name" example:"Test"`

Reply:
Thanks @Nazarii-4chain good finding - but this model is unused (we only use OldPaymailAddress model) so I remove redundant one.